### DISCUSSION TITLE
Pipe Poiseuille Flow - Differences between DNS-LES-VLES

### DISCUSSION THREAD

**Discussion by AlejandroEngESC** [firemodels/fds#13606] [Q&A] [answered]
Dear community, 

I've been running FDS simulations using Pyrosim 2023 preprocessor (and FDS 6.8.0 solver) for pipe flows without an initial fire, just an inlet-outlet condition, with a mass supply at the inlet using a "supply" conditionb, whereas at the outlet is set as "open" condition. 
I've noticed that when using VLES CFD code the results doesn't show the typical Poiseuille flow (parabolic velocity distribution) with zero velocity at the walls. This condition is achieved when using LES or DNS CFD code, with the same ammount of elements for the 3 simulations.

The model is attached.

&HEAD CHID='Tinlet_110_Tagua_20'/
&TIME T_END=40.0/
&DUMP DT_RESTART=300.0, DT_SL3D=0.25/
&MISC GVEC=-1.0E-10,-1.0E-10,-9.81, TMPA=110.0, POROUS_FLOOR=.FALSE./
&PART ID='Water',
      SPEC_ID='Water drops',
      DIAMETER=10.0,
      MONODISPERSE=.TRUE.,
      AGE=60.0,
      SAMPLING_FACTOR=1,
      INITIAL_TEMPERATURE=20/
&PART ID='Particle02',
      SPEC_ID='Water drops',
      DIAMETER=50.0,
      MONODISPERSE=.TRUE.,
      AGE=60.0,
      INITIAL_TEMPERATURE=99/

&PROP ID='Generic Industrial Spray',
      PART_ID='Water',
      PARTICLES_PER_SECOND=20000,
      FLOW_RATE=25.0,
      FLOW_RAMP='Generic Industrial Spray_FLOW_RAMP',
      ORIFICE_DIAMETER=3.0E-3,
      SPRAY_ANGLE=1.0,3.0/
&RAMP ID='Generic Industrial Spray_FLOW_RAMP', T=0.0, F=0.0/
&RAMP ID='Generic Industrial Spray_FLOW_RAMP', T=20.0, F=0.0/
&RAMP ID='Generic Industrial Spray_FLOW_RAMP', T=21.0, F=1.0/
&RAMP ID='Generic Industrial Spray_FLOW_RAMP', T=60.0, F=1.0/
&DEVC ID='NOZZLE', PROP_ID='Generic Industrial Spray', XYZ=1.689311,0.305198,0.317634, ORIENTATION=1.0,0.0,0.0, QUANTITY='TIME', SETPOINT=0.0/
&SURF ID='Supply',
      RGB=26,204,26,
      TMP_FRONT=110.0,
      MASS_FLUX=11.87248,10.41436,8.95585,7.4989,
      SPEC_ID='ISO-OCTANE','N-HEPTANE','N-HEXANE','N-PENTANE',
      TAU_MF=-4.0,-4.0,-4.0,-4.0/

&VENT ID='Outlet', SURF_ID='OPEN', XB=40.715265,40.715265,-0.30538,1.0,-0.33698,1.0/ 
&VENT ID='Inlet', SURF_ID='Supply', XB=0.0,0.0,-0.30538,1.0,-0.33698,1.0/ 

&BNDF QUANTITY='AMPUA', PART_ID='Water'/
&BNDF QUANTITY='MPUA', PART_ID='Water'/
&BNDF QUANTITY='AMPUA_Z', SPEC_ID='WATER VAPOR'/
&BNDF QUANTITY='MPUA_Z', SPEC_ID='WATER VAPOR'/

My question are:
1- What should be changed between solvers to achieve the same solution in all of them?
2- Why is the reason the VLES solver is not able to replicate a Poisuille flow at the pipe whereas the LES/DNS are able without need of mesh refinement? Note: All 3 simulations take around the same amount of running time. 
3- What are the assumptions when ussing LES and DNS to be able to achieve a "realistic" solution when the mesh size is relatively large?

Thank you in advance for your time and your fast reply.

**Comment by mcgratta** [comment]
Open up the User's Guide and search on the term VLES, DNS, SVLES, etc. That should answer most of your questions.

**Reply by AlejandroEngESC** [reply]
Hello, 

Thank you for your fast reply. I've already open the User's Guide before posting this question. The question was posted since no clear answer was obtained. Anyways, thank you again for your advice!

**Reply by johodges** [reply, marked as answer]
Take a look at section 10.1.7 - tangential velocity boundary conditions at solid surfaces. If you want to have true no slip at the wall set NO_SLIP=T
